.travis.yml

```diff
@@ -3,8 +3,12 @@ sudo: required
 install:
   - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
 script:
+  - cp qmp.opam opam
```

Comment:
Why do we need two differently-named copies of this file?
Would a symlink work?

Reply:
This is just in the Travis machine and just to make the scripts happy. I don't think this is worth sweating about.